docs/adr/0012-output-markdown-format.md

```diff
@@ -0,0 +1,58 @@
+# 12. Output Markdown Format
+
+Date: 2021.09.09
+
+## Status
+
+proposed
+
+## Context
+
+Stencil has an output target that is capable of generating markdown documentation for the components that it compiles.
+Markdown does not have a unifying standard, which has led to multiple different implementations of how the markup is
+parsed. This has resulted in various markdown renderers whose implementations yield different renderings. Depending on
+where an individual views a markdown file, the results may look incorrect. 
+
+There was a [recent pull request](https://github.com/ionic-team/stencil/pull/3042/) that the team ultimately decided to
+close for not following CommonMark, which is the proposed/intended solution. Stencil should formally decide on a 
+proposed standard and use that for guidance as to how markdown should be formatted.
+
+## Options
+
+1. [Describe the many options, their pros/cons, side effects, and any perspective a given team member has]
+2. Adopt [CommonMark](https://commonmark.org/)
+   1. Pros:
+      1. It has industry support from the likes of GitHub, GitLab, Qt, and more
+      2. It seems to be the leading effort to create an unambiguous specification for Markdown
+      3. Using a well adopted standard may lead to less chance of Markdown renderings not working as intended
+   2. Cons
+      1. Not everyone may be familiar with CommonMark, leading to confusion when reviewing pull requests
+      2. CommonMark has yet to produce a v1.0 specification
+3. Adopt [GitHub Flavored Markdown (GFM)](https://www.iana.org/assignments/markdown-variants/GFM)
+   1. Pros:
+      1. Individuals may be more familiar with GFM, given the popularity of the platform
+      2. Using a well adopted standard may lead to less chance of Markdown renderings not working as intended
+   2. Cons:
+      1. GFM is less defined compared to CommonMark, resulting in potential ambiguities
```

Comment:
Modern GFM is well-defined and a [strict superset of CommonMark](https://github.blog/2017-03-14-a-formal-spec-for-github-markdown/). However, the [current GFM spec](https://github.github.com/gfm/) is based on version 0.29 of CommonMark while the latest version of [CommonMark](https://spec.commonmark.org/) is 0.30.

Reply:
TIL - I was looking at an old IANA doc. Fixed in 675f1d4075dde8c7c264d07ec42cb8255c3b5ea9